docs/variants/pc_engines/initial-deployment.md

```diff
@@ -0,0 +1,85 @@
+# Initial deployment
+
+This document describes the process of updating
+[PC Engines coreboot firmware](https://pcengines.github.io/) to Dasharo firmware
+based on EDK II.
+
+## OS compatibility
+
+Switching from BIOS to UEFI-based firmware might cause compatibility issues if
+you already had an operating system installed on your platform. Some OSs can
+handle the switch without any issues, while others may need to be reinstalled.
+We have tested the update on several operating systems. The results are
+presented in the table below:
+
+| OS | Can be booted from UEFI  |
+|----|------------------------- |
+| Ubuntu | :heavy_check_mark:   |
+| OPNSense | :heavy_check_mark: |
+| pfSense | :x:                 |
+| OpenWrt | Depends[^1]         |
```

Comment:
Don't understand why we are OpenWrt OS is listed here? We are not adding support for additional OS in this release. `Depends` is not acceptable test result, but I think we should remove OpenWrt support.

Reply:
> Don't understand why we are OpenWrt OS is listed here? 

Because it was on the list of OSes we wanted to see if they still can boot after firmware transition. `Depends` is linked to the explanation what it means, feel free to suggest a better word here if you have it in mind.